csrc/kernel_cache.h

```diff
@@ -209,6 +225,9 @@ class TORCH_CUDA_CU_API FusionKernelRuntime {
   //! Utility to speed up value evaluation at runtime
   std::unique_ptr<PrecomputedValues> precomputed_values_;
 
+  //! Cache of all tensors in the complete fusion
```

Comment:
Ok, thank you for explanation.

Reply:
Thanks for checking the scope / meaning of complete fusion. If complete fusion were to mean fusion for segment then it would have require more fine tune handling but hopefully this is not the case. Now it is clear.

Thanks!